WeakAuras/WeakAuras.lua

```diff
@@ -4008,15 +4010,15 @@ end
 
 -- handle trigger updates that have been requested to be sent into custom
 -- we need the id and triggernum that's changing, but can't send the ScanEvents to the custom trigger until after UpdatedTriggerState has fired
-local trigger_to_custom_Queue = {}
+local TTC_Queue = {}
 function Private.AddTriggerToCustomQueue(id, triggernum)
-  trigger_to_custom_Queue[id] = triggernum
+  TTC_Queue[id] = triggernum
```

Comment:
This does not support multiple triggers in queue for same aura

Reply:
It would never need to. Perhaps "queue" isn't quite the right word. The only intention is to delay sending to custom until the previous state is fully updated.